pallets/living-assets-ownership/src/tests.rs

```diff
@@ -1,137 +1,100 @@
+		));
+		assert_eq!(LivingAssetsModule::collection_base_uri(0).unwrap(), base_uri);
 	});
 }
 
 #[test]
-fn living_assets_ownership_trait_create_new_collection_should_emit_an_event() {
+fn create_new_collections_should_emit_events_with_collection_id_consecutive() {
 	new_test_ext().execute_with(|| {
 		// Go past genesis block so events get deposited
 		System::set_block_number(1);
 
-		assert_ok!(<LivingAssetsModule as CollectionManager<AccountId>>::create_collection(ALICE));
+		assert_ok!(LivingAssetsModule::create_collection(
+			RuntimeOrigin::signed(ALICE),
+			"ciao".into()
+		));
 		System::assert_last_event(Event::CollectionCreated { collection_id: 0, who: ALICE }.into());
+		assert_ok!(LivingAssetsModule::create_collection(
+			RuntimeOrigin::signed(ALICE),
+			"ciao".into()
+		));
+		System::assert_last_event(Event::CollectionCreated { collection_id: 1, who: ALICE }.into());
+		assert_ok!(LivingAssetsModule::create_collection(
+			RuntimeOrigin::signed(ALICE),
+			"ciao".into()
+		));
+		System::assert_last_event(Event::CollectionCreated { collection_id: 2, who: ALICE }.into());
+		assert_ok!(LivingAssetsModule::create_collection(
+			RuntimeOrigin::signed(ALICE),
+			"ciao".into()
+		));
+		System::assert_last_event(Event::CollectionCreated { collection_id: 3, who: ALICE }.into());
 	});
 }
 
 #[test]
-fn living_assets_ownership_trait_id_of_new_collection_should_be_consecutive() {
-	new_test_ext().execute_with(|| {
-		assert_eq!(
-			<LivingAssetsModule as CollectionManager<AccountId>>::create_collection(ALICE).unwrap(),
-			0
-		);
-		assert_eq!(
-			<LivingAssetsModule as CollectionManager<AccountId>>::create_collection(ALICE).unwrap(),
-			1
-		);
-		assert_eq!(
-			<LivingAssetsModule as CollectionManager<AccountId>>::create_collection(ALICE).unwrap(),
-			2
-		);
-		assert_eq!(
-			<LivingAssetsModule as CollectionManager<AccountId>>::create_collection(ALICE).unwrap(),
-			3
-		);
-		assert_eq!(
-			<LivingAssetsModule as CollectionManager<AccountId>>::create_collection(ALICE).unwrap(),
-			4
-		);
-		assert_eq!(
-			<LivingAssetsModule as CollectionManager<AccountId>>::create_collection(ALICE).unwrap(),
-			5
-		);
-	});
-}
-
-#[test]
-fn erc721_owner_of_asset_of_unexistent_collection() {
-	new_test_ext().execute_with(|| {
-		let result = <LivingAssetsModule as Erc721>::owner_of(0, 2.into());
-		assert_err!(result, traits::Erc721Error::UnexistentCollection);
-	});
-}
-
-#[test]
-fn erc721_owner_of_asset_of_collection() {
+fn create_collection_with_base_uri_greater_than_limit() {
 	new_test_ext().execute_with(|| {
-		let collection_id =
-			<LivingAssetsModule as CollectionManager<AccountId>>::create_collection(ALICE).unwrap();
+		let base_uri: Vec<u8> = vec![0; 255 + 1]; // TODO: use BaseURILimit::get() + 1
 		assert_eq!(
-			<LivingAssetsModule as Erc721>::owner_of(collection_id, 2.into()).unwrap(),
-			H160::from_low_u64_be(0x0000000000000002)
+			LivingAssetsModule::create_collection(RuntimeOrigin::signed(ALICE), base_uri)
```

Comment:
I suggest using `assert_noop!` macro instead as it ensures also that there were no changes in the storage runtime